spock-specs/src/test/groovy/org/spockframework/smoke/mock/JavaStubs.groovy

```diff
@@ -141,6 +141,24 @@ class JavaStubs extends Specification {
     person.phoneNumber == "6789"
   }
 
+  def "can stub final methods with mockito with closure"() {
+    FinalMethodPerson person = Stub(mockMaker: MockMakers.mockito) {
```

Comment:
Unfortunetely, this specs is not consitent in its use, but new features should include proper block labels.

```suggestion
    given:
    FinalMethodPerson person = Stub(mockMaker: MockMakers.mockito) {
```